src/core/data-cite.js

```diff
@@ -52,11 +54,18 @@ function requestLookup(conf) {
     }
     switch (elem.localName) {
       case "a": {
+        if (elem.text === "") {
+          elem.text = title;
```

Comment:
Did you intend to use `textContent` here?

Reply:
Agree, textContent would be better.

Reply:
```suggestion
          elem.textContent = title;
```